locales-pending/onboarding.ftl

```diff
@@ -47,6 +47,7 @@ onboarding-enter-details-placeholder-last-name = Enter last name
 onboarding-enter-details-label-location = City and state
 onboarding-enter-details-placeholder-location = Enter city and state
 onboarding-enter-details-label-date-of-birth = Date of birth
+onboarding-enter-details-placeholder-date-of-birth = Enter date of birth
```

Comment:
Hmm, so I'm not sure if there's a better option in this particular case, since we don't know what the exact format of the displayed date is going to be, but at least for educational value: the `placeholder` attribute [is supposed to](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#placeholder) hint to what type of data should go in there, rather than explaining what the field is.

>  So for example if a field is expected to capture a user's first name, and its label is "First Name", a suitable placeholder might be "e.g. Mustafa".

It also doesn't mention `date` as one of the valid elements it can go on though, so that's an extra argument for why it doesn't matter much in this specific case :)

Reply:
That makes sense. Yeah, showing the exact format as a placeholder gets tricky. I guess if we are OK with not showing a placeholder that should be the easiest solution for now.